org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/outline/OutlinePanelView.java

```diff
@@ -1149,7 +1149,7 @@ protected void initForm()
     final EnabledTest notNarrative = getNotNarrativeTest();
     final EnabledTest notIsLayer = getNotLayerTest();
 
```

Comment:
@saulhidalgoaular  & @helenayele  - I don't know how this typo fell through the net 🤣

Reply:
oohhh...